frontend/src/components/UserTables/UnscheduledTable.tsx

```diff
@@ -18,17 +18,18 @@ function renderTableHeader() {
 }
 
 const Table = () => {
+  const [openModal, setOpenModal] = useState(-1);
```

Comment:
Just want to clarify... the number here will correspond to the index in passengers array below?

Reply:
Yep, it basically corresponds to the row number because if I just have a bool then all of the dropdowns for all buttons open at once--is there a better way to do this?

Reply:
I think this might be the best way. I was just making sure I understood the code.